docs/system-overview.md

```diff
@@ -0,0 +1,77 @@
+A separate component - the simulator - will handle all controll the simulation, provide external interfaces and store what is happening.
+
+! There will be also something that provides a video feed for livestreaming the simulation. 
+  Since it is still unclear how this will work this is not included in this description.
+
+![System overview](images/system-overview.png)
+
+## Autonomous Systems.
+
+Every autonomous system run on their own seperated environment. 
+Idealy this would be separate virtual machines but also Docker containers could be used.
+The autonomous systems are expected to continuously run a ROS master.
+When the simulator is ready to do a drive for the given autonomous system, the smulator will launch a ros node connected to this autonomous system's ros master.
+This ros node will exist outside of the autonomous system's environment, instead it will run on the simulator computer.
+The simulator ros node will publish sensordata and listen for car setpoints on a set of predifned topics.
+When the simulation is finished, the ros node will disconnect from the autonomous system's ros master and sensordata is no longer streaming.
+
+During the competition the teams will not be allowed to access their autonomous systems.
+All remote access to the environments will be cut off completely.
+So every autonomous system must be able to run multiple missions without human interferance.
+To let the simulation know what is expected from it, the simulator will send mission controll messages a few seconds before the event start.
+These messages contain information about the mission (trackdrive, autocross, etc).
+When the autonomous system receives a mission message it can expect to receive sensordata shortly after.
+Since the laptimer for all events start whenever the car crosses the start-line, the autonomous system's can take all the time they need to launch their relevant algorithms for the mission (within reasonable time).
+
+## The Simulator
+
+The simulator is a continuously running program that is like the spider in the web.
+
+The simulator it offers a webinterface to the event's officials. 
+Using this webinterface the officials can choose what team/car is going to drive and which track is going to be used.
+The official will also be able to send the start signal, view lap times, down or out cones, car off course's.
+There is also an emergency stop button in case the car is uncontrollable.
+
+The simulator connects to the simulated world inside Unreal Engine using AirSim (more on that later).
+It also connects to the autonomous systems and acts like a bridge between the two.
+On one hand it requests sensordata and passes it along on ros topics to the current autonomous system.
+On the other hand it receives car controll commands from the autonomous system and forwards it to the virtual world.
+
+Only 1 car will be able to run on this autonomous system.
```

Comment:
Only 1 car at a time

The wording makes it sound like only 1 car will race for the whole competition ;)